lib/Runtime/Library/JavascriptNumber.h

```diff
@@ -185,7 +185,7 @@ namespace Js
     // Since RecyclableObject and Int32 in their Var representation have top 14 bits 0
     // and tagged float now has top 14 XOR'd with 1s - the value with 14 bits non zero is
     // a float.
-    // A float can have all 14 bits 0 iff it was a NaN in the first place. Since
+    // A float can have all 14 bits 0 if it was a NaN in the first place. Since
```

Comment:
ohh looked like a typo instead. wanted to fix it on the way. 

reverted this back. PR is updated.